src/data/movies.js

```diff
@@ -0,0 +1,202 @@
+        "vote_count": 5349,
+        "release_date": "2017-10-25",
+        "poster_path": "https://image.tmdb.org/t/p/w500/rzRwTcFvttcN1ZpX2xv4j3tSdJu.jpg",
+        "overview": "Thor is imprisoned on the other side of the universe and finds himself in a race against time to get back to Asgard to stop Ragnarok, the prophecy of destruction to his homeworld and the end of Asgardian civilization, at the hands of an all-powerful new threat, the ruthless Hela.",
+        "budget": 180000000,
+        "revenue": 854229371,
+        "genres": [
+            "Action",
+            "Adventure",
+            "Fantasy"
+        ],
+        "runtime": 130
+    },
+    {
+        "id": 321612,
+        "title": "Beauty and the Beast",
+        "tagline": "Be our guest.",
+        "vote_average": 6.8,
+        "vote_count": 7861,
+        "release_date": "2017-03-16",
+        "poster_path": "https://image.tmdb.org/t/p/w500/tWqifoYuwLETmmasnGHO7xBjEtt.jpg",
+        "overview": "A live-action adaptation of Disney's version of the classic tale of a cursed prince and a beautiful young woman who helps him break the spell.",
+        "budget": 160000000,
+        "revenue": 1263521126,
+        "genres": [
+            "Family",
+            "Fantasy",
+            "Romance"
+        ],
+        "runtime": 129
+    },
+    {
+        "id": 399055,
+        "title": "The Shape of Water",
+        "tagline": "A Fairy Tale for Troubled Times",
+        "vote_average": 7.3,
+        "vote_count": 3200,
+        "release_date": "2017-12-01",
+        "poster_path": "https://image.tmdb.org/t/p/w500/k4FwHlMhuRR5BISY2Gm2QZHlH5Q.jpg",
+        "overview": "An other-worldly story, set against the backdrop of Cold War era America circa 1962, where a mute janitor working at a lab falls in love with an amphibious man being held captive there and devises a plan to help him escape.",
+        "budget": 19500000,
+        "revenue": 185545281,
+        "genres": [
+            "Drama",
+            "Fantasy",
+            "Romance"
+        ],
+        "runtime": 123
+    },
+    {
+        "id": 141052,
+        "title": "Justice League",
+        "tagline": "",
+        "vote_average": 6.4,
+        "vote_count": 3960,
+        "release_date": "2017-11-15",
+        "poster_path": "https://image.tmdb.org/t/p/w500/eifGNCSDuxJeS1loAXil5bIGgvC.jpg",
+        "overview": "Fuelled by his restored faith in humanity and inspired by Superman's selfless act, Bruce Wayne and Diana Prince assemble a team of metahumans consisting of Barry Allen, Arthur Curry and Victor Stone to face the catastrophic threat of Steppenwolf and the Parademons who are on the hunt for three Mother Boxes on Earth.",
+        "budget": 500000000,
+        "revenue": 655894816,
+        "genres": [
+            "Action",
+            "Adventure",
+            "Fantasy",
+            "Science Fiction"
+        ],
+        "runtime": 120
+    }
+];
+
+const moviesMapped = movies.map(movie => {
+    return {
```

Comment:
Hi Natalia,
all fields except **release_date** are the same, would it be better to use Object.assign? (Not critical)